tests/compiler/test_plugins.py

```diff
@@ -255,3 +257,25 @@ def test_plugin_load_exception(snippetcompiler):
     )
     with pytest.raises(CompilerException, match=re.escape(expected)):
         compiler.do_compile()
+
+
+def test_3457_helpful_string(snippetcompiler, caplog):
+    with caplog.at_level(logging.DEBUG):
+        snippetcompiler.setup_for_snippet(
+            """
+entity A:
+end
+A.other [0:] -- A
+implement A using std::none
+a = A()
+std::attr(a, "other")
+a.other = A()
+            """
+        )
+        compiler.do_compile()
+    dir: str = snippetcompiler.project_dir
+    message: str = (
+        "Unset value in python code in plugin at call: inmanta_plugins.std::attr "
```

Comment:
`inmanta_plugins.std::attr` is a mix of Python and DSL, which is not ideal. This should be either `inmanta_plugins.std.attr` or `std::attr`. The latter would be most appropriate in my opinion. Let me know if you want me to have a look on where to get the inmanta module corresponding to this Python module.

Reply:
i'll have a look tomorrow and i'll let you now :)